source/blog/2018-06-08-the-emberjs-times-issue-50.md

```diff
@@ -0,0 +1,104 @@
+The RFC to **deprecate global resolver** is has now entered the final stage. The motivation of this RFC is the fact that we have transitioned to using Ember CLI. The globals resolver is primarily used for Ember without Ember CLI.
+
+Read all about the RFC and bring your thoughts for the final process in the [RFC repository](https://github.com/emberjs/rfcs/pull/331).
+
+---
+
+## [Ember CLI Packager: News from the Field 🌾](https://github.com/ember-cli/rfcs/blob/master/active/0051-packaging.md)
+
+We already wrote about the [Packager in Ember CLI recently](https://www.emberjs.com/blog/2018/05/25/the-emberjs-times-issue-48.html#toc_a-href-https-github-com-ember-cli-ember-cli-pull-7826-a-package-out-for-delivery-to-ember-cli-a). But here's another **status update** for this
+feature which will make the **build pipeline** of your Ember apps **easier to modify** very soon:
+These weeks lots of new improvements and fixes landed ([1](https://github.com/ember-cli/ember-cli/pull/7864), [2](https://github.com/ember-cli/ember-cli/pull/7855), [3](https://github.com/ember-cli/ember-cli/pull/7848), [4](https://github.com/ember-cli/ember-cli/pull/7860)).
+According to internal sources, there have already been **successful test runs** using the experimental version of this [Rollup](https://rollupjs.org/guide/en) powered packager 😲 for bundling and tree-shaking
+different open-source Ember 🐹 projects.
+We'll keep you posted with more news from the field soon!
+
+---
+
+## [SECTION TITLE](#section-url)
+
+---
+
+## [Got Something to Share?](https://airtable.com/shrDbeo2Y80OPG0kC)
+We all love reading the Ember blog. It’s an awesome way to find out what’s happening in the community and to keep up to date on all things Ember. Sometimes though, you may think to yourself, “I’ve got something to say!”. Awesome! Whether your passion is authoring addons or tinkering with broccoli, **contributing to the blog** is an excellent opportunity to connect with the community. Well, getting started could not be any easier! If you want to write a guest blog post for the website, make your way over to [Ember.js Guest Blog Post Invitation](https://airtable.com/shrDbeo2Y80OPG0kC), and fill out the form!
+
+---
+
+## [The Shortest Ember Book](https://github.com/ember-learn/the-shortest-ember-book)
+
+<div class="blog-row">
+  <img class="float-left small transparent padded" alt="Shortest Ember Book" title="Shortest Ember Book" src="/images/about/shortest-ember-book.jpg" />
+
+  <p>The Shortest Ember Book is [here](https://github.com/ember-learn/the-shortest-ember-book)! The goal is to quickly and concisely communicate what Ember is, how it works, and steps to get started. Anyone is welcome to print the book! Meetup organizers, speakers, and conference table hosts are especially encouraged to print and distribute copies. Shoutout to [Braden Lawrence](http://www.bradenlawrence.com/portfolio) for the designs and [Jen Weber](https://github.com/jenweber) for leading the project.</p>
```

Comment:
I think these links have to become `<a>` tags unfortunately, because the block is wrapped in HTML already to make the floating image work